front_end/test/test.js

```diff
@@ -0,0 +1,41 @@
+const Application = require('spectron').Application
+const assert = require('assert')
+const electronPath = require('electron')
+const path = require('path')
+const app = require('electron').remote
+
+
+describe("Application launch test", function () {
+  this.timeout(20000)
+	
+	beforeEach(function () {
+		
+    const appPath = path.join(__dirname, "../");
+
+		  this.app = new Application({
```

Comment:
This file could be formatted better to be more consistent with the rest of the code.